tests/functional/test_cli.py

```diff
@@ -19,6 +19,7 @@ def setUp(self):
 
         self.source_dir = tempfile.mkdtemp()
         self.artifacts_dir = tempfile.mkdtemp()
+        self.scratch_dir = "/tmp/scratch"
```

Comment:
windows

Reply:
I would expect that to fail in appveyor through.